dev/merge_zeppelin_pr.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2
```

Comment:
This will not work in all the cases. For example this is what I see on my local.
```
python             python2.7          python3            python3.6          python3.6m         pythonw            
python-config      python2.7-config   python3-config     python3.6-config   python3.6m-config  pythonw2.7
```

And now when I try to run this it gives me an error `/usr/bin/python2: bad interpreter: No such file or directory`